.github/workflows/Tests.yml

```
@@ -58,3 +58,34 @@ jobs:
       run: php artisan migrate
     - name: Execute tests (Unit and Feature tests) via PHPUnit/Pest
       run: php artisan test --parallel
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)